blocks/cards/cards.js

```diff
@@ -3,12 +3,10 @@ import { domEl } from '../../scripts/dom-helpers.js';
 
 export default function decorate(block) {
   block.classList.add('calcite-mode-dark');
+  block.classList.add('cardsperrow');
 
   const processSimpleCard = (div) => {
-    if (!block.classList.contains('simple')) {
-      return;
-    }
-    block.classList.add('cardsperrow');
+    if (!block.classList.contains('simple')) return;
```

Comment:
Instead I'd suggest only caling this function when the card is "simple" instead (and same for standardCard).

Reply:
Updated